container/src/LuigiContainer.svelte

```diff
@@ -121,6 +121,10 @@
 
   const initialize = (thisComponent: any) => {
     if (!containerInitialized) {
+      if (!sandboxRules && thisComponent.getAttribute('sandboxRules')) {
+        sandboxRules = JSON.parse(thisComponent.getAttribute('sandboxRules'));
+      }
+
```

Comment:
I was expecting only added Tests in this PR. How come we need to add these lines. 

The **sandboxRules** feature was already added previously [here](https://github.com/SAP/luigi/pull/3650).

What do these lines help with? 

If this is a discovered bug, then a Bug PR for this would be more than welcome. Its much easier to look back at Git history when Reviewing in the future, but also helps spread the word that there was some issue with  this feature at some point in time.